src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -129,14 +129,15 @@ public void assertEquals(int expectedCode) {
      * @param expectedJson - the expected response json object
      */
     public void assertEquals(JsonObject expectedJson) {
-        Success success = Success.FAIL;
+        boolean pass = false;
         if (object != null) {
-            success = object.equals(expectedJson) ? Success.PASS : Success.FAIL;
+            pass = object.equals(expectedJson) ? true : false;
+        }
+        if (pass) {
+            reporter.pass("", "Expected to find a response of: " + reporter.formatResponse(new Response(0, expectedJson, null)), FOUND + reporter.formatResponse(this));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Expected to find a response of: " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)